app/src/main/java/org/p2p/wallet/home/events/AppLoader.kt

```diff
@@ -1,7 +1,14 @@
 package org.p2p.wallet.home.events
 
-interface AppLoader {
-    suspend fun onLoad()
-    suspend fun onRefresh(): Unit = Unit
-    suspend fun isEnabled(): Boolean = true
+abstract class AppLoader {
+    private val dependencies: MutableList<AppLoader> = mutableListOf()
+    abstract suspend fun onLoad()
+    open suspend fun onRefresh(): Unit = Unit
+    open suspend fun isEnabled(): Boolean = true
```

Comment:
I would rather make it as abstract to make loaders more explicit - you clearly see conditions when they are loading and you won't forget to use FT

Reply:
Upd: I see, they are unique in the Facade